site/public/js/server.js

```diff
@@ -1096,6 +1096,12 @@ function enableTabsInTextArea(jQuerySelector) {
     });
 }
 
+function confirmBypass(str, redirect){
+    confirmed = confirm(str);
+    if (confirmed)
+        location.href = redirect;
```

Comment:
Submitty's stye guide requires parentheses on all if-statements.  It's also generally better practice, since future developers will have to edit fewer lines if they want to add more lines to the body of the statement.

```suggestion
    if (confirmed) {
        location.href = redirect;
    }
```